pkgs/applications/misc/keepassx/reboot.nix

```diff
@@ -1,23 +1,23 @@
 { stdenv, fetchFromGitHub, cmake, libgcrypt, qt5, zlib, libmicrohttpd, libXtst }:
 
 stdenv.mkDerivation rec {
-  name = "keepassx2-http-unstable-${version}";
-  version = "2016-05-27";
+  name = "keepassx-reboot-${version}";
+  version = "2.0.3-http";
```

Comment:
Hi. Last time it was pointed out to me, that we shouldn't use these namespaces (I assume you're coming from arch too :wink: ) See. https://github.com/NixOS/nixpkgs/pull/17778#discussion-diff-75496293 by @RamKromberg 

I suggest we omit the ´-http´ since the packagename is different.